app/migrations/Version20250402103942.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20250402103942 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE form_fields ADD field_width VARCHAR(255) DEFAULT \'100%\' NOT NULL');
```

Comment:
I've executed the migration and then run `bin/console d:s:u --dump-sql | grep width` to see if Doctrine finds any differences between the Entity definition and the alter table executed in the migration. The result is 

```
ALTER TABLE form_fields CHANGE field_width field_width VARCHAR(50) DEFAULT '100%' NOT NULL;
```

So there is some misalignment. I can se the varchar length. So I did this change:

```suggestion
        $this->addSql('ALTER TABLE form_fields ADD field_width VARCHAR(50) DEFAULT \'100%\' NOT NULL');
```

And then run the migration again. The diff is then empty so I think that's it.